starter-code/index.js

```diff
@@ -1,10 +1,50 @@
-var SortedList = function() {};
 
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+var SortedList = function() {
+    this.length = 0;
+    this.items = [];
+};
+//PUSHES ITEM, SORTS ALL ITEMS IN ARRAY, UPDATES LENGTH PROPERTY//
+SortedList.prototype.add     = function(item) {
+    this.items.push(item);
+    this.items.sort(function(a, b){return a - b}); 
+    this.length = this.items.length;
+};
+//RETURN Nth ITEM(-1 differenence between NUM & ARRAY)
+SortedList.prototype.get     = function(pos) {
+    return this.items[pos - 1];
+};
+//MAX
+SortedList.prototype.max     = function() {
+    return Math.max.apply(null, this.items);
+}
+//MIN
+SortedList.prototype.min     = function() {
```

Comment:
Very clever using the `.apply` method here. Just so you can compare, here was another way to write this method:
```js
  var minNumber =
    this.list.reduce((minSoFar, number) => {
      return Math.min(minSoFar, number);
    });
  return minNumber;
};
```